mutable_tree_test.go

```diff
@@ -1443,52 +1456,309 @@ func TestMutableTree_InitialVersion_FirstVersion(t *testing.T) {
 	_, err := tree.Set([]byte("hello"), []byte("world"))
 	require.NoError(t, err)
 
+	// More than 1 key/value pair to also check non-root nodes
+	_, err = tree.Set([]byte("goodbye"), []byte("world"))
+	require.NoError(t, err)
+
 	_, version, err := tree.SaveVersion()
 	require.NoError(t, err)
 	require.Equal(t, initialVersion, version)
 
+	// err = tree.ndb.batch.WriteSync()
+	// require.NoError(t, err)
+
+	// Save 2 empty versions after the initial version, should produce 2 refnodes
+	_, _, err = tree.SaveVersion()
+	require.NoError(t, err)
+
+	_, latestVersion, err := tree.SaveVersion()
+	require.NoError(t, err)
+
+	// ------------------------------
+	// Verify
+	// 1. Node exists with Version 1, not InitialVersion
+	// 2. VersionExists(InitialVersion) - should return true
+	// 3. AvailableVersions() - should include InitialVersion and NOT version 1
+	// 4. ndb.GetRoot(InitialVersion) - should return the root node
+	// 5. GetImmutable(InitialVersion) - should return the immutable tree
+	// 6. LoadVersion(InitialVersion) - loads tree even if node contains version 1. This is last since it runs some previous methods internally
+
 	// the nodes created at the first version are not assigned with the `InitialVersion`
-	rootKey := GetRootKey(1)
-	node, err := tree.ndb.GetNode(rootKey)
+	t.Run("1. root node exists at RootKey(initialVersion) with nodeKey version 1", func(t *testing.T) {
+		t.Skip()
```

Comment:
Is the `t.Skip()` old?  I think this test should be updated to assert that GetNode(GetRootKey(1)) does not return a node?

Reply:
ah yes ty I forgot to update this 😄